src/js/13-cookies.js

```diff
@@ -0,0 +1,126 @@
+const name = 'plenty-cookies'
+
+class Cookies {
+  constructor () {
+    this.addListeners()
+    if (document.cookie.indexOf(name) === -1) {
+      this.toggleCookies()
+    }
+    this.setDefaultSettings()
+  }
+
+  addListeners () {
+    const acceptAllButton = document.getElementsByClassName('cookie-accept')
+    const acceptSelectedButton = document.getElementsByClassName('cookie-accept-selection')
+    const declineButton = document.getElementsByClassName('cookie-deny')
+    const showfurtherSettings = document.getElementById('show-cookie-further-settings')
+    const hidefurtherSettings = document.getElementById('hide-cookie-further-settings')
+
+    acceptAllButton[0].addEventListener('click', () => { this.setCookie(true, true, true, true) })
+    acceptSelectedButton[0].addEventListener('click', () => {
+      this.setCookie(this.consent, this.statistics.googleAnalytics, this.media.youtube, this.media.vimeo, this.functional.preferredLanguage)
+    })
+    declineButton[0].addEventListener('click', () => { this.setCookie(false, false, false, false) })
```

Comment:
```suggestion
    declineButton[0].addEventListener('click', () => { this.setCookie(true, false, false, false) })
```
Shouldn't the first one be set to true, so that the necessary cookies are always active?

Reply:
@sene-plenty @pfrincu-plenty 
Double-checked and always setting the essential cookies to `true` is the desired behaviour.

Reply:
done